e2e/cypress/e2e/sidebar/quickstart-sidebar.cy.ts

```diff
@@ -0,0 +1,415 @@
+/**
+ * This test suite validates that sidebar filtering and tagger menu functionality works in the fiftyone app with quickstart dataset.
+ */
+
+const selectSamplesInGrid = (indices: number[]) => {
+  indices.forEach((i) => {
+    cy.get("[data-cy=looker]")
+      .eq(i)
+      .within(() => {
+        cy.get("input").click({ force: true });
+      });
+  });
+};
+
+describe("Sidebar filter", () => {
+  context("quickstart dataset filter", () => {
+    before(() => {
```

Comment:
Should we add an after to cleanup the dataset?

Reply:
https://docs.cypress.io/guides/references/best-practices#State-reset-should-go-before-each-test

In `cypress/support/e2e.ts`, we clean datasets in the `before()` hook:

```
// delete all datasets before each spec runs
before(() => {
  cy.executePythonCode(
    `
    import fiftyone as fo
    fo.delete_datasets("*")
    `
  );
});
```